modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageLeaderElectionListener.java

```diff
@@ -94,7 +98,8 @@ public class MetaStorageLeaderElectionListener implements LeaderElectionListener
             CompletableFuture<MetaStorageServiceImpl> metaStorageSvcFut,
             ClusterTimeImpl clusterTime,
             CompletableFuture<MetaStorageConfiguration> metaStorageConfigurationFuture,
-            List<ElectionListener> electionListeners
+            List<ElectionListener> electionListeners,
+            BooleanSupplier leaderSecondaryDutiesPaused
```

Comment:
Can you give a description somewhere of what this supplier is for?

Reply:
Added a description to the field